yarn.lock

```diff
@@ -4858,12 +4858,12 @@ is-string@^1.0.5:
   resolved "https://registry.yarnpkg.com/is-string/-/is-string-1.0.5.tgz#40493ed198ef3ff477b8c7f92f644ec82a5cd3a6"
   integrity sha512-buY6VNRjhQMiF1qWDouloZlQbRhDPCebwxSjxMjxgemYT46YMd2NR0/H+fBhEfWX4A/w9TBJ+ol+okqJKFE6vQ==
 
-is-svg@^3.0.0:
-  version "3.0.0"
-  resolved "https://registry.yarnpkg.com/is-svg/-/is-svg-3.0.0.tgz#9321dbd29c212e5ca99c4fa9794c714bcafa2f75"
-  integrity sha512-gi4iHK53LR2ujhLVVj+37Ykh9GLqYHX6JOVXbLAucaG/Cqw9xwdFOjDM2qeifLs1sF1npXXFvDu0r5HNgCMrzQ==
+is-svg@^3.0.0, is-svg@^4.3.1:
```

Comment:
Should version 3.0.0 be removed from this list?

Reply:
Unfortunately this is how yarn lock works when you selectively resolve a dependency.

Reply:
>If one of the dependencies has a new version available and the available version is within the specified version range in the package.json, then the newest dependency will be installed.

Reply:
https://robertcooper.me/post/how-yarn-lock-files-work-and-upgrading-dependencies

Reply:
LGTM